scripts/emerge/ingest_to_tdr.py

```diff
@@ -180,26 +183,24 @@ def create_recoded_json(row_json):
     return recoded_row_json
 
 
-def parse_json_outputs_file(input_tsv, bucket, target_table_name, dataset_id):
-    """Format the json file containing workflow outputs and headers."""
+def parse_json_outputs_file(input_tsv):
+    """Create a recoded json dictionary per row in input."""
 
     tsv_df = pd.read_csv(input_tsv, sep="\t")
+    all_recoded_row_dicts = []
 
-    basename = input_tsv.split(".tsv")[0]
-    output_filename = f"{basename}_recoded_newline_delimited.json"
-
-    all_rows = []
     for index, row in tsv_df.iterrows():
-        # create timestamp
+        # create output filename
```

Comment:
is this comment accurate? i don't see a filename being created

Reply:
ah yeah it used to exist there but i moved around a few things and left that behind